docs/source/examples/array-random.rst

```diff
@@ -0,0 +1,11 @@
+Creating random arrays
+======================
+
+We can create arrays with random variables using ``da.random.normal``.
```

Comment:
Suggest the following language:

> In a simple case we create arrays with random data using the ``da.random`` module.

For the following reasons

1.  The term random variable is a bit special and doesn't apply here.  Instead we have normally distributed random data
2.  We don't give the impression that you have to use `da.random.normal` to create random data, but rather there is a full module to choose from